C7FIT/ScheduleViewController.swift

```diff
@@ -41,4 +55,149 @@ class ScheduleViewController: UIViewController {
+                return cell
+            }
+        } else if indexPath.row == 2 {
+            if let cell: ScheduleContactTableViewCell = tableView.dequeueReusableCell(withIdentifier: "ContactCell") as? ScheduleContactTableViewCell  {
+                cell.contactButton.addTarget(self, action: #selector(self.contactButtonPressed), for: .touchUpInside)
+                return cell
+            }
+        }
+        return UITableViewCell()
+    }
+    
+    // MARK: - Cell Functions
+    // MARK: BrowserCell Link Function
+    
+    func scheduleLinkPressed() {
+        print("scheduleLinkPressed")
+        if let linkUrl = URL(string: "http:clubsevenfitness.com") {
+            if #available(iOS 10, *) {
+                UIApplication.shared.open(linkUrl, options: [:],
+                                          completionHandler: {
+                                            (success) in
+                                            print("Open \(success)")
+                })
+            } else {
+                let success = UIApplication.shared.openURL(linkUrl)
+                print("Open \(success)")
+            }
+        }
+    }
+    
+    
+    //MARK: ContactCell Button Function
+    func contactButtonPressed() {
+        print("contact button pressed")
+        let contactGymAlert: UIAlertController = UIAlertController(title: "Contact Us", message: nil, preferredStyle: .actionSheet)
+        let cancelAction: UIAlertAction = UIAlertAction(title: "Cancel", style: .cancel) {
+            action -> Void in
+        }
+        let emailAction: UIAlertAction = UIAlertAction(title: "Email", style: .default, handler: { action in
+            print("email")
+            self.sendEmail()
+        })
+        let textAction: UIAlertAction = UIAlertAction(title: "Text", style: .default, handler: { action in
+            print("text")
+            self.sendText()
+        })
+        let callAction: UIAlertAction = UIAlertAction(title: "Call", style: .default, handler: { action in
+            print("Call")
+            self.callGym()
+        })
+        
+        contactGymAlert.addAction(cancelAction)
+        contactGymAlert.addAction(emailAction)
+        contactGymAlert.addAction(textAction)
+        contactGymAlert.addAction(callAction)
+
+        self.present(contactGymAlert, animated:true, completion:nil)
+    }
+    
+    func sendEmail() {
+        if MFMailComposeViewController.canSendMail() {
+            let gymEmail = "infinitenl@gmail.com"
+            let sampleBody = "<p>Testing text</p>"
+            let mail = MFMailComposeViewController()
+            mail.mailComposeDelegate = self
+            mail.setToRecipients([gymEmail])
+            mail.setMessageBody(sampleBody, isHTML: true)
+            
+            present(mail, animated: true, completion: nil)
+        } else {
+            print(MFMailComposeViewController.canSendMail())
```

Comment:
can display a message to the user here and tell them that they need to setup their email on their phone for this to work